lib/src/git.rs

```diff
@@ -238,11 +247,11 @@ pub fn export_refs(
     let all_local_branch_names: HashSet<&str> = view
         .git_refs()
         .keys()
-        .filter_map(|git_ref| git_ref.strip_prefix("refs/heads/"))
+        .filter_map(|r| ref_name_to_local_branch_name(r))
```

Comment:
nit: can `|r| ref_name_to_local_branch_name(r)`be just `ref_name_to_local_branch_name`?

Reply:
This gives me an error, surprisingly.

```
error[E0631]: type mismatch in function arguments
   --> lib/src/git.rs:250:21
    |
54  | fn ref_name_to_local_branch_name(ref_name: &str) -> Option<&str> {
    | ---------------------------------------------------------------- found signature defined here
...
250 |         .filter_map(ref_name_to_local_branch_name)
    |          ---------- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected due to this
    |          |
    |          required by a bound introduced by this call
    |
    = note: expected function signature `fn(&std::string::String) -> _`
               found function signature `for<'a> fn(&'a str) -> _`
```

Not my favorite side of Rust.